pkg/plugins/golang/v3/scaffolds/internal/templates/config/prometheus/monitor.go

```diff
@@ -53,6 +53,10 @@ spec:
   endpoints:
     - path: /metrics
       port: https
+      scheme: https
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tlsConfig:
+        insecureSkipVerify: true
```

Comment:
The webhook server already [sets up TLS](https://github.com/kubernetes-sigs/controller-runtime/blob/774f9d490254c861e9f5ca009b3c4b8b5d55fa83/pkg/webhook/server.go#L175). I wonder if that cert/key can be used here.

Reply:
This seems out of scope of this PR, but seems like a good idea otherwise.